app/assets/stylesheets/base/_base.scss

```diff
@@ -0,0 +1,4 @@
+.container {
+  margin: 1rem;
+  width: fit-content;
+}
```

Comment:
Add new line at the end of this file please

Reply:
done